rest/data/test.go

```diff
@@ -39,26 +39,10 @@ func FindTestById(id string) ([]testresult.TestResult, error) {
 }
 
 func (tc *DBTestConnector) FindTestsByTaskId(opts FindTestsByTaskIdOpts) ([]testresult.TestResult, error) {
-	t, err := task.FindOneIdNewOrOld(opts.TaskID)
-	if err != nil {
-		return []testresult.TestResult{}, gimlet.ErrorResponse{
-			StatusCode: http.StatusNotFound,
-			Message:    errors.Wrapf(err, "finding task '%s'", opts.TaskID).Error(),
-		}
-	}
-	if t == nil {
```

Comment:
tests seem to be failing because we dont check the task exists i think?

Reply:
I'll take a look! FWIW there's no need for reviewers to debug why tests might be failing, that falls on the author. LGTMs always have the caveat that it's for when tests pass. If enough things are failing that it doesn't seem worth reviewing yet you can always wait until the author has fixed them to review (and/or comment something like that)